packages/angular_devkit/build_angular/src/builders/server/platform-server-exports.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+export * from '@angular/platform-server';
```

Comment:
@AndrewKushnir, maybe we can be more selective here and export what we need.

Reply:
I like the idea of being more conservative and just export what we need. At least, we can do that from the very beginning and change this later if needed (if this becomes an issue and we end up updating this constantly). If this is a viable solution overall, I can update https://github.com/angular/angular/pull/47103 to add a new token that we re-export here.

Meanwhile, would it be possible to export our "rendering" functions from here, such as `renderModule` and `renderApplication` (for standalone use-cases), so we don't need changes to the `main.server.ts` to proxy the `renderApplication` to make it available to NgUniversal?

Reply:
Yeah that is viable. 

To export `renderModule` we'd need to do a migration to remove it from the `main.server.ts` exports to avoid colliding exports. 

For the `renderApplication`, that is correct given that users **will never** need to set `platformProviders` themselves. If there are cases were users would need to set such providers than a "proxy" is the the only way forward. 

// cc @clydin to see if he has any objections to this change.

Reply:
sounds good.  I definitely prefer the minimal approach to start.